src/main/java/org/apache/sysml/api/jmlc/Connection.java

```diff
@@ -63,6 +54,15 @@
 import org.apache.sysml.runtime.util.DataConverter;
 import org.apache.wink.json4j.JSONObject;
 
+import java.io.BufferedReader;
+import java.io.Closeable;
+import java.io.FileReader;
+import java.io.IOException;
+import java.io.InputStream;
+import java.io.InputStreamReader;
+import java.util.HashMap;
+import java.util.Map;
```

Comment:
Will do. I guess, we are conforming to eclipse style imports. This is the intellij style imports.
I followed [this](http://stackoverflow.com/questions/14716283/is-it-possible-for-intellij-to-organize-imports-the-same-way-as-in-eclipse) guide. 
We should quickly decide on a code style that we can all conform to and have all our IDEs use this style file.